Jakefile.js

```diff
@@ -973,6 +973,8 @@ var servicesLintTargets = [
 });
 var lintTargets = compilerSources
     .concat(harnessCoreSources)
+    // Other harness sources
+    .concat(["instrumenter.ts"].map(f => path.join(harnessDirectory, f)))
     .concat(serverCoreSources)

```

Comment:
Looks like `=>` is not supported on the CI server. So it should be `function (f) { ... }`